docs/entities/entity-attack.md

```diff
@@ -0,0 +1,490 @@
+
+```
+          "entity_types": [
+            {
+              "filters": {
+                "any_of": [
+                  { "test": "is_family", "subject": "other", "operator": "==", "value": "snow_golem" },
+                  { "test": "is_family", "subject": "other", "operator": "==", "value": "iron_golem" }
+                ]                //anything that is equal to either "snow_golem" or "iron_golem"
+              },
+              "max_dist": 24
+            },
+            {
+              "filters": {
+                "all_of": [
+                  { "test": "is_family", "subject": "other", "operator": "==", "value": "player" },
+                  { "test": "has_equiptment", "subject": "other", "domain": "head", "operator": "=!", "value": "turtle_helmet" }
+                ]                //anything equal to player AND not wearing "turtle_helmet" on head
+              },
+              "max_dist": 24
+            }
+          ]
+```
+This would only target `snow_golem`s, `iron_golem`s, and `player`s that are **not** wearing `turtle_helmet`s.
+
+
+### Melee
+
+Melee attacks are the most common type of attack, they cause knockback, and have a 100% success rate at accuracy.
+
+```
+      "wiki:melee_attack": {
+        "minecraft:attack": {                  //defines the base stats of melee attacks
+          "damage": 3,
+          "effect_name": "slowness",
+          "effect_duration": 20
+        },
+        "minecraft:behavior.melee_attack": {   //defines the properties of the melee attack
+          "priority": 3,
+          "speed_multiplier": 1,
+          "track_target": false,
+          "require_complete_path": true
+        }
+      },
+```
+Set the damage, choose a mob effect, and change some additional properties.
+
+The value defined in components stating integers of damage can simply be a constant, or a string containing 2 numbers, for a range of possible values.
+
+`"damage": 3` would result in 3 each time
+
+`"damage": [ 2, 6 ]` would result in any integer between 2 and 6
+
+Both the mob effect and duration timer are optional, but when they are used, the available effects are as following:
+
+| Effect Name     |
+| --------------- |
+| speed           |
+| slowness        |
+| haste           |
+| mining_fatigue  |
+| strength        |
+| instant_health  |
+| instant_damage  |
+| jump_boost      |
+| nausea          |
+| regeneration    |
+| resistance      |
+| fire_resistance |
+| water_breathing |
+| invisibility    |
+| blindness       |
+| night_vision    |
+| hunger          |
+| weakness        |
+| poison          |
+| wither          |
+| health_boost    |
+| absorbtion      |
+| saturation      |
+| levitation      |
+| fatal_poison    |
+| slow_falling    |
+| conduit_power   |
+| bad_omen        |
+| village_hero    |
+| darkness        |
+
+
+### Ranged
+
+Fires specified [projectiles](/docs/documentation/projectiles) towards target at set intervals.

```

Comment:
@Luthorius here is your first dead link. It points to a site that doesn't exist. You need to remove the `/docs` part, because it should be relative and not absolut.